### DISCUSSION TITLE
Mounts, pets, animal companions

### DISCUSSION THREAD

**Discussion by Odoamar** [fmasa/wfrp-master#107] [General]
Do you have plans to add a similar field to app? My plater already bought a horse and I don't know where she can put information about this.

**Comment by fmasa** [comment]
This is currently somewhat supported as one player can have multiple Characters.

This feature is intended for various "Character" types:
- Familiars
- Hirelings
- Mounts

I think the case can be made for distinguishing between these types internally, which would allow PCs to choose and ride a mount.

**Comment by Odoamar** [comment]
Yea, It covers all my needs.
